lib/cassandra_migrations/cql-rb-wrapper.rb

```diff
@@ -48,6 +47,7 @@ def use(keyspace)
     if @sessions[keyspace]
       @session = @sessions[keyspace]
     else
+      Rails.logger.try(:info, "Creating Cassandra session: #{keyspace.inspect}")
```

Comment:
That's ok. It is good to have some logging. Since we have when the session is opened this already serves the purpose.